web-api/migration-terraform/main/lambdas/migration.js

```diff
@@ -32,7 +32,7 @@ const processItems = async ({ documentClient, items, migrateRecords }) => {
 
 const getFilteredGlobalEvents = event => {
   const { Records } = event;
-  return Records.map(item =>
+  return Records.filter(item => item.eventName !== 'REMOVE').map(item =>
```

Comment:
Confirming my understanding here — this would mean that records which were deleted _while a migration is running_ are not removed from the destination table, leaving orphaned records?

Reply:
I think that sounds correct.  I think that was missing logic which we need to support.  This PR is to fix a bug that is blocking dynamodb modifications / inserts.

I think we need to figure out a race condition because if the migration is running, and a user deletes a record, that record might be re-added via the migration script at a later time.  Open to suggestions

Reply:
Thanks, Cody!